tests/boardfarm_plugins/boardfarm_hotfix.patch

```diff
@@ -0,0 +1,34 @@
+SPDX-License-Identifier: BSD-2-Clause-Patent
+SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+This code is subject to the terms of the BSD+Patent license.
+See LICENSE file for more details.
+
+diff --git a/boardfarm/lib/bft_logging.py b/boardfarm/lib/bft_logging.py
+index 109fab0..ebb3f9d 100644
+--- a/boardfarm/lib/bft_logging.py
++++ b/boardfarm/lib/bft_logging.py
+@@ -238,9 +238,9 @@ def create_file_logs(config, board, tests_to_run, logger):
+                     line = line[1:]
+                 if line.startswith(' ['):
+                     line = line[1:]
+-                ts, text = line.split(']', 1)
++                text = line.split(']', 1)
+                 combined_list.append({
+-                    "time": float(ts[1:-1]),
++                    "time": 0,
+                     "text": str(text),
+                     "name": name
```

Comment:
This doesn't seem right - I think it will replace one bug with another one. I guess the problem was that this would do the wrong thing in case `line` didn't contain the ` [<timestamp>]` part. So I guess the correct fix would be:
```
if line.startswith(' ['):
    line = line[1:]
    ts, text = line.split(']', 1)
    timestamp = float(ts[1:-1])
else:
    line = text
    timestamp = 0.0

combined_list.append({
    "time": timestamp,
    "text": str(text),
...
```

Reply:
> This doesn't seem right - I think it will replace one bug with another one. I guess the problem was that this would do the wrong thing in case `line` didn't contain the ` [<timestamp>]` part. So I guess the correct fix would be:
> 
> ```
> if line.startswith(' ['):
>     line = line[1:]
>     ts, text = line.split(']', 1)
>     timestamp = float(ts[1:-1])
> else:
>     line = text
>     timestamp = 0.0
> 
> combined_list.append({
>     "time": timestamp,
>     "text": str(text),
> ...
> ```

Added that solution to PR:
https://github.com/prplfoundation/boardfarm/pull/8